spacy/tests/lang/es/test_noun_chunks.py

```diff
@@ -1,6 +1,148 @@
+from spacy.tokens import Doc
 import pytest
 
 
+# fmt: off
+@pytest.mark.parametrize(
+    "words,heads,deps,pos,chunk_offsets",
+    [
+        # un gato -> "un gato"
+        (
+            ["un", "gato"],
+            [1, 1],
+            ["det", "ROOT"],
+            ["DET", "NOUN"],
+            [(0, 2)],
+        ),
+        # la camisa negra -> "la camisa negra"
+        (
+            ["la", "camisa", "negra"],
+            [1, 1, 1],
+            ["det", "ROOT", "amod"],
+            ["DET", "NOUN", "ADJ"],
+            [(0, 3)],
+        ),
+        # un lindo gatito -> "un lindo gatito"
+        (
+            ["Un", "lindo", "gatito"],
+            [2, 2, 2],
+            ["det", "amod", "ROOT"],
+            ["DET", "ADJ", "NOUN"],
+            [(0,3)]
+        ),
+        # una chica hermosa e inteligente -> una chica hermosa e inteligente
+        (
+            ["Una", "chica", "hermosa", "e", "inteligente"],
+            [1, 1, 1, 4, 2],
+            ["det", "ROOT", "amod", "cc", "conj"],
+            ["DET", "NOUN", "ADJ", "CCONJ", "ADJ"],
+            [(0,5)]
+        ),
+        # Tengo un gato y un perro -> un gato, un perro
+        ( 
+            ["Tengo", "un", "gato", "y", "un", "perro"],
+            [0, 2, 0, 5, 5, 0],
+            ["ROOT", "det", "obj", "cc", "det", "conj"],
+            ["VERB", "DET", "NOUN", "CCONJ", "DET", "NOUN"],
+            [(1,3), (4,6)]
+         
+        ),
+        # Dom Pedro II -> Dom Pedro II
+        (
+            ["Dom", "Pedro", "II"],
+            [0, 0, 0],
+            ["ROOT", "flat", "flat"],
+            ["PROPN", "PROPN", "PROPN"],
+            [(0,3)]
+        ),
+        # los Estados Unidos -> los Estados Unidos
+        (
+            ["los", "Estados", "Unidos"],
+            [1, 1, 1],
+            ["det", "ROOT", "flat"],
+            ["DET", "PROPN", "PROPN"],
+            [(0,3)]
+        ),
+        # Miguel de Cervantes -> Miguel de Cervantes
+        (
+            ["Miguel", "de", "Cervantes"],
+            [0, 2, 0],
+            ["ROOT", "case", "flat"],
+            ["PROPN", "ADP", "PROPN"],
+            [(0,3)]
+        ),
+        (
+            ["Rio", "de", "Janeiro"],
+            [0, 2, 0],
+            ["ROOT", "case", "flat"],
+            ["PROPN", "ADP", "PROPN"],
+            [(0,3)]
+        ),
+        # la destrucción de la ciudad -> la destrucción, la ciudad
```

Comment:
I have a question: I understand we want all the noun chunks here but "a destrucción de la ciudad" is a complex noun phrase in itself. We're not interested in marking those as noun chunks as well? Another example of what I have in mind is "The cat that looks nice". Would that phrase just have one noun chunk "The cat"?

Reply:
Sure, this is a point I thought about as well. Here, I inspired from how we did for English. For English the NP  `destruction of the city` results in two small NPs `destruction` and `the city`. There are 2 ways to do it here, either a long NP or 2 NPs. I chose the way we do it for English (same for Portuguese). What do you think? If you think the whole NP is more useful for users, I can change the PR a little bit, definitely no problem.

Reply:
I agree. It'd be better to have all languages work in a similar wat if possible. Also we always know what the smallest noun phrases will look like. The complex ones are unbounded so maybe they should be parsed elsewhere if parsed at all, I think.

Reply:
@DuyguA Yes that's how I'd expect it to be :)